include/sea_dsa/DsaColor.hh

```diff
@@ -0,0 +1,126 @@
+#ifndef __DSA_COLOR_HH_
+#define __DSA_COLOR_HH_
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/GraphTraits.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Module.h"
+
+#include "sea_dsa/BottomUp.hh"
+#include "sea_dsa/CallSite.hh"
+#include "sea_dsa/Global.hh"
+#include "sea_dsa/Graph.hh"
+#include "sea_dsa/GraphTraits.hh"
+
+using namespace sea_dsa;
+using namespace llvm;
+
+class Color {
+private:
+  int m_r, m_g, m_b;
+
+public:
+  Color();
+  Color(int r, int g, int b);
+
+  void randColor();
+  std::string stringColor() const;
+};
+
+enum e_color {WHITE, BLACK, GRAY}; // colors for exploration
+
+
+typedef DenseMap<const Cell*, e_color> ExplorationMap;
```

Comment:
Use the `using` keyword from C++11 instead of typedef

Reply:
Fixed